owl/EASE-SAY.owl

```diff
@@ -113,6 +184,89 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ContainmentTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ContainmentTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#ControlTheory"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">GUM-4-space: A functional relation describing that the location of some entity, the containee, is constrained to be within some region which is the interior of some other entity, the container.
+
+This is also known as FunctionalControlInternal in GUM.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ControlTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ControlTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#FunctionalSpatialSchemaTheory"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">GUM-4-space: The binary functional relation holding between two spatial objects x and y, such that x controls y&apos;s position in space; if x moves, then y moves as well; x and y need not be in contact.</rdfs:comment>
```

Comment:
what about roles in this theory?

Reply:
It inherits axioms from the parent. Basically, the same roles that are important for the superclass are important here as well and there is no need to have the same axioms twice.